Frontend/ui-customer/index.js

```diff
@@ -228,6 +228,10 @@ window["onEvent'"] = function (event, args) {
     purescript.onConnectivityEvent("LOCATION_DISABLED")();
   } else if (event == "onInternetChanged") {
     purescript.onConnectivityEvent("INTERNET_ACTION")();
+  } else if (event == "onKeyboardOpen" || event == "onKeyboardClose") {
```

Comment:
`else if ((event == "onKeyboardOpen" || event == "onKeyboardClose") && window.keyBoardCallback)`
Simplify this